src/LibVLCSharp.Forms/Shared/TrackViewModel.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace LibVLCSharp.Forms.Shared
+{
+    /// <summary>
+    /// Defines the <see cref="TrackViewModel" />.
+    /// </summary>
+    public class TrackViewModel : INotifyPropertyChanged
+    {
+        /// <summary>
+        /// Track description Id
+        /// </summary>
+        public int Id { get; set; }
```

Comment:
It's not an id, it's an index, not to be confused with the track id in the VLC side.
Can you explain again why the VLC track id doesn't fit our need?

Reply:
If we can't use the VLC track id, then we should:
- Change the name here to avoid confusion
- Document in the comments why it's not the same number